src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -677,6 +708,23 @@ The next four parameters are optional, and apply to advanced Azure endpoint conf
 - *privateNetworkCIDR:* An optional network Classless Inter-Domain Routing to allocate the new VMs within the private network. The default value is _10.0.0.0/24_.
 - *staticPublicIP:* A boolean flag to determine whether the public IPs of the infrastructure's VMs should be static. The default value is _true_.
 - *additionalProperties:* Additional JVM properties to configure your ProActive node. The default values allow to handle both PNP and PAMR protocols. You can add your own properties but be aware that removing the current values might cause the deployment to fail.
+- *linuxStartupScript:* Linux VM startup script to launch the ProActive nodes. 
+    Usually, this script is expected to download and install the Java Runtime Environment, then download and launch ProActive node.jar, if not already provided by the VM image.
+    The script can use <<_cloud_infrastructure_startup_script,some arguments>>. For example, it can use `%nodeJarUrl%` to represent the full URL path for downloading the _node.jar_ file. These arguments will be interpreted by Azure Infrastructure later.
+    This is an optional property. If left blank, the script is automatically generated. Here is the default value of linuxStartupScript:
+
+        mkdir -p /tmp/node && cd /tmp/node
+        if ! type -p jre/bin/java; then wget -nv -N https://s3.amazonaws.com/ci-materials/Latest_jre/jre-8u131-linux-x64.tar.gz; tar -xf jre-8u131-linux-x64.tar.gz; mv jre1.8.0_131/ jre; fi
+        wget -nv %nodeJarUrl%
+        nohup jre/bin/java -jar node.jar -Dproactive.communication.protocol=%protocol% -Dpython.path=%jythonPath% -Dproactive.pamr.router.address=%rmHostname% -D%instanceIdNodeProperty%=%instanceId% -r %rmUrl% -s %nodeSourceName% %nodeNamingOption% -v %credentials% -w %numberOfNodesPerInstance% %additionalProperties% &
+
+- *windowsStartupScript:* Powershell script to be run during Windows VM startup for launching the ProActive nodes.
+    This is an optional property. If left blank, the script is automatically generated. Here is the default value of windowsStartupScript:
+
+        mkdir -p /tmp/node && cd /tmp/node
```

Comment:
Doesn't look a powershell script to me, Does it ?

Reply:
Great finding. I copy-pasted the wrong code.